### PR TITLE
Complete redirection

### PR DESCRIPTION
**Pull request by ZhizhenQin**
Summary: This modification redirects users to the data visualization page from the home screen.

Test plan: Start the server, go to the home page, and check that the page is redirected.

### CODE REVIEW

File: ailab/ailab/views.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from django.shortcuts import redirect
+
+def redirect_to_viz(request):
+    response = redirect('/benchmark/visualize')
+    return response
```
Comment (sf-wind) [inline ailab/ailab/views.py]: Please add a new line after this

File: ailab/ailab/views.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+from django.shortcuts import redirect
```
Comment (sf-wind) [inline ailab/ailab/views.py]: Please add the follow line above this line:

from __future__ import absolute_import, division, print_function, unicode_literals

File: ailab/ailab/views.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+
+def redirect_to_viz(request):
```
Comment (sf-wind) [inline ailab/ailab/views.py]: one more blank line please.

File: ailab/ailab/views.py [hunk 4]
```diff
@@ -0,0 +1,7 @@
+from future import absolute_import, division, print_function, unicode_literals
```
Comment (sf-wind) [inline ailab/ailab/views.py]: The lint actually suggested the following... sorry about it

from __future__ import absolute_import, division, print_function, unicode_literals

Comment (sf-wind) [reply ailab/ailab/views.py]: I see. github automatically removes the double underlines. It should be:
`from __future__ import absolute_import, division, print_function, unicode_literals`

Comment (ZhizhenQin) [reply ailab/ailab/views.py]: Sure. Will pay attention in the future.

**Pull request closed without merging**